.travis.yml

```diff
@@ -33,8 +33,18 @@ matrix:
           - tox --installpkg ./dist/djangorestframework-*.whl
           - tox  # test sdist
 
+      - python: "3.7"
+        name: "Docs broken link check"
+        script:
+          - pip install -r requirements/requirements-documentation.txt
+          - pip install pylinkvalidator
```

Comment:
I would go ahead and add this to the `requirements/requirements-documentation.txt`

Reply:
I updated the implementation , but I'm not able to use tox.

```
script:
          - pip install requirements/requirements-documentation.txt
          - mkdocs serve &
          - WAIT_TIME=0 && until nc -vzw 2 localhost 8000 || [ $WAIT_TIME -eq 5 ]; do sleep $(( WAIT_TIME++ )); done
          - if [ $WAIT_TIME == 5 ]; then echo cannot start mkdocs server on http://localhost:8000; exit 1; fi
          - pylinkvalidate.py -P http://localhost:8000/
```

The `mkdocs serve &` is supposed to start and run in the background, which works fine in the regular Bash command line, but not in the tox `commands`. Is it possible to achieve this? It seems that the `&` is escaped: 

- https://travis-ci.org/nezhar/django-rest-framework/jobs/596736576
- https://travis-ci.org/nezhar/django-rest-framework/jobs/596773933

I also tried running it as a `commands_pre` without the `&`, but as expected this is blocking the pipeline: https://travis-ci.org/nezhar/django-rest-framework/jobs/596743816

Reply:
Thanks for attempting this. In retrospect, we probably don't need to add this to tox, since it will be uncommon task to run.